iosApp/Survey/Sources/Presentation/Modules/Splash/SplashView.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  SplashView.swift
+//  Survey
+//
+//  Created by Bliss on 26/9/22.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+import FlowStacks
+import SwiftUI
+
+protocol SplashCoordinator {
+
+    func showLogin()
+}
+
+struct SplashView: View {
+
+    @State var animating = false
+
+    let coordinator: SplashCoordinator
+
+    private let animationDuration: Double = 0.7
+
+    var body: some View {
+        ZStack {
+            Assets.background
+                .image
+                .resizable()
+                .aspectRatio(contentMode: .fill)
+                .ignoresSafeArea()
+                .frame(maxWidth: .infinity, maxHeight: .infinity)
+
+            if animating {
+                Assets.backgroundBlur
+                    .image
+                    .resizable()
+                    .aspectRatio(contentMode: .fill)
+                    .ignoresSafeArea()
+                    .frame(maxWidth: .infinity, maxHeight: .infinity)
+            }
+
+            VStack(
+                alignment: .center
+            ) {
+                Assets.logoWhite.image
+
+                if animating {
+                    Spacer().frame(maxHeight: 300.0)
+                }
+            }
+            .padding(.horizontal, 24.0)
+        }
+        .accessibility(.splash(.view))
+        .onAppear {
+            withAnimation(.linear(duration: animationDuration)) {
+                animating = true
+            }
+            DispatchQueue.main.asyncAfter(deadline: .now() + animationDuration - 0.1) {
+                withAnimation(.linear(duration: 0.2)) {
+                    coordinator.showLogin()
+                }
+            }
+        }
```

Comment:
I do think the Splash screen is duplicated to the [Login screen](https://github.com/blyscuit/kmm-swiftUI-IC/blob/ba7a44e89315166d5395bb7a07b2fecbad91fd7a/iosApp/Survey/Sources/Presentation/Modules/Login/LoginView.swift). From the design preview, do you think we should add animation on the Login screen to switch between Splash and Login views? 

There is an `animation gap` between Splash and Login screens from your POW 💭 

https://user-images.githubusercontent.com/6356137/192242284-94dc29d1-6aa2-454f-ab58-ca0ef2eee5ec.mp4

Reply:
Makes more sense to do animation on the Login screen 👍

Reply:
Update POW.